build.gradle.kts

```diff
@@ -1,17 +1,12 @@
 /*
- * Copyright 2023 Responsive Computing, Inc.
+ * Copyright 2024 Responsive Computing, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * This source code is licensed under the Responsive Software License Agreement v1.0 available at
```

Comment:
This should be "This source code is licensed under the Responsive Business Source License v1.0 available at.."